### PR TITLE
Mobile version milestone

### PR DESCRIPTION
**Pull request by MaiRizk**
# Work and about me sections

#### The project has been updated for this milestone with the following updates;

1. Enhance styles to exactly match the design.
2. Add the work section to contain examples for different projects.
-- Each project in a separate card --
3. About me section that contains;
- Brief introduction about myself.
- Social links for connections.
- ul that contains languages, frameworks, and skills.
- Languages list.

*Styles are done exactly as the design without any functionality as requested in this milestone.*

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -26,46 +26,246 @@
...
+					About <br />
+					Myself
+				</h2>
+				<p class="intro">
+					Hello I’m a software developer! I can help you build a product ,
+					feature or website Look through some of my work and experience! If you
+					like what you see and have a project you need coded, don’t hestiate to
+					contact me.
+				</p>
+				<p class="primary connect">LET’S CONNECT</p>
```
Comment (levy002) [inline index.html]: - [ ] Kindly make sure to use the `hx` (heading) tag while this `LET'S CONNECT` is a title, it is better if we use a proper tag for presenting the title which makes our works look more professional and increase our page accessibility. :wink:

File: styles.css [hunk 2]
```diff
@@ -62,6 +73,149 @@ h1 {
...
+  color: #396df2;
+  font-size: 17px;
+  font-weight: 500;
+  background: white;
+  border: 1px solid #396df2;
+  border-radius: 8px;
+  padding: 12px 13px;
+  margin-bottom: 12px;
+  font-family: "Poppins", Arial, Helvetica, sans-serif;
+}
```
Comment (levy002) [inline styles.css]: - [ ] You did a great ajob in styling the `Get resume` button however, please make to implement the button interactions design `Active`, `Hover` and `Pressed` `Disabled`.
  
    - Kindly, take a look to your Figma design and implement the required button interactions
       
    image

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hi @MaiRizk,
### I'm Levy, your PR reviewer

Awesome work, you've done :sunglasses: 
There are some issues that you still need to work on to go to the next project but you are almost there!

You can do it 

## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Highlights :muscle: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your README file is professional :heavy_check_mark: 
- Correct usage of Github flow :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Awesome styling :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by awais-amjed — APPROVED** [review]
Hi @MaiRizk , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Great work on implementing the required changes 👏 Congratulations! 🎉🥇✨

## Highlights

- [x] Descriptive Pull Request ✔️
- [x] Nice Design ✔️
- [x] Good code organization ✔️ 

gif

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**